Makefile

```diff
@@ -0,0 +1,3 @@
+npC: src/forth.c
+	gcc -Wall -std=c99 src/forth.c src/gui.c src/logic.c src/graphics.c src/global.c src/utils.c src/stack.c `sdl2-config --libs --cflags` -lm -lncurses -o Final.out
+
```

Comment:
Usually `Final.out` is not something one would expect as the entry point of a program. If you changed it consider an update to the README.md file as well.